### PR TITLE
(docs): automatically add Supported Version to README

### PR DESCRIPTION
**Pull request by bakerboy448**
## Description

- update README with the supported qbit version in master and develop based on Qbittorrent-api in requirements.txt

Fixes # (issue)

## Type of change

- docs

## Checklist:

- [ ] test

### CODE REVIEW

**Comment by bakerboy448** [comment]
image

File: README.md [hunk 1]
```diff
@@ -23,21 +25,38 @@ This is a program used to manage your qBittorrent instance such as:
...
-2. Once installed, you have to [set up your Configuration](https://github.com/StuffAnThings/qbit_manage/wiki/Config-Setup) by create a [Configuration File](https://github.com/StuffAnThings/qbit_manage/blob/master/config/config.yml.sample) filled with all your values to connect to your qBittorrent instance.
-3. Please refer to the list of [Commands](https://github.com/StuffAnThings/qbit_manage/wiki/Commands) that can be used with this tool.
+1. Install qbit_manage either by installing Python 3.8.1+ on the localhost and following the [Local Installation](https://github.com/StuffAnThings/qbit_manage/wiki/Local-Installations) Guide or by installing Docker and following the [Docker Installation](https://github.com/StuffAnThings/qbit_manage/wiki/Docker-Installation) Guide or the [unRAID Installation](https://github.com/StuffAnThings/qbit_manage/wiki/Unraid-Installation) Guide.
+1. Once installed, you have to [set up your Configuration](https://github.com/StuffAnThings/qbit_manage/wiki/Config-Setup) by create a [Configuration File](https://github.com/StuffAnThings/qbit_manage/blob/master/config/config.yml.sample) filled with all your values to connect to your qBittorrent instance.
```
Comment (bobokun) [inline README.md:47]: Are these changes intentional? the numbering all changing to 1.

Comment (bakerboy448) [reply README.md:47]: Markdown formats it numerically

**Review by bobokun — APPROVED** [review]
